.circleci/main.yml

```diff
@@ -592,6 +637,41 @@ jobs:
       - run:
           name: Push docker
           command: make docker-push VERSION=${CIRCLE_SHA1}
+  build-docker-images-with-coverage:
```

Comment:
any way we could simplify this instead of using a near duplicate of same job. maybe a conditional parameter. you also might want to rebase the branch. i updated goreleaser, and it not 1.10.3 anymore.

Reply:
Done. I haven't rebased yet, but I thought I did pretty recently, I'll double check before merge.